docs/verify/simulate-with-qiskit-primitives.mdx

```diff
@@ -2,263 +2,243 @@
+<Tabs>
+  <TabItem value="V2" label="StatevectorEstimator (V2)">
 
-If you prefer to define a new [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) with new options, define a `dict` like this one:
+      The sampling overhead is now called `precision`, and is defined exclusively at the `.run()`
+      method level. This allows for a more finely-grained tuning of the option all the way to the PUB level.
 
-```python
-options = {"shots": 2048, "seed": 123}
-```
+    ```python
+    # Circuits and parameters must obey the Instruction Set Architecture (ISA) of a particular backend.
+    from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
 
-You can then introduce it into your new [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) with the `options` argument.
+    pm = generate_preset_pass_manager(backend=backend, optimization_level=1)
+    isa_circuit1 = pm.run(circuit1)
+    isa_circuit2 = pm.run(circuit2)
 
-```python
-estimator = Estimator(options=options)
+    # Estimate expectation values for two PUBs, both with 0.05 precision.
+    estimator.run([(isa_circuit1, obs_array1), (isa_circuit2, obs_array_2)], precision=0.05)
+    ```
 
-job = estimator.run(qc, observable)
-result = job.result()
-print(result)
-```
+  </TabItem>
 
-```python
-EstimatorResult(values=array([4.]), metadata=[{'variance': 3.552713678800501e-15, 'shots': 2048}])
-```
+  <TabItem value="V1" label="Estimator (V1)">
 
-```python
-print(result.values[0])
-```
+      The V1 interface specifies shots in the following ways:
 
-```python
-3.999999998697238
-```
-## Compute circuit output probabilities with `Sampler` primitive
+        - Setting keyword arguments in the [`qiskit.primitives.Estimator.run`](../api/qiskit/qiskit.primitives.Estimator#run) method.
+        - Modifying the [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) options.
 
-Follow these instructions to get the probability distribution of a quantum circuit with the [`qiskit.primitives.Sampler`](../api/qiskit/qiskit.primitives.Sampler) primitive.
+    ```python
+    # Circuits and parameters must obey the Instruction Set Architecture (ISA) of a particular backend.
+    from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
 
-<Admonition type="note">
-  While this guide uses Qiskit’s reference implementation, the `Sampler` primitive can be run with any provider using [`qiskit.primitives.BackendSampler`](../api/qiskit/qiskit.primitives.BackendSampler).
+    pm = generate_preset_pass_manager(backend=backend, optimization_level=1)
+    isa_circuit = pm.run(qc)
 
-  ```python
-  from qiskit.primitives import BackendSampler
-  from <some_qiskit_provider> import QiskitProvider
+    job = estimator.run(isa_circuit, observable, shots=2048, seed=123)
+    result = job.result()
+    print(result)
```

Comment:
Missing output

Reply:
Added e331fa0c1c4cabfc0afdeac21644b2c6deebb28d but does it look right?

Reply:
Becky recommends deleting the print(result) line since the V2 version doesn't have output, so keeping the V1 parallel